pyproject.toml

```diff
@@ -6,3 +6,163 @@ requires = [
     "pybind11>=2.6",
     "setuptools_scm>=7",
 ]
+[tool.ruff]
```

Comment:
Done, this was the way it was because I just did `flake8-to-ruff >> pyproject.toml` and made sure it worked, didn't hand edit style too much outside of the per file ignores.